modules/test/services/python/src/services_module.py

```diff
@@ -196,10 +196,9 @@ def _process_port_results(self):
       self._scan_results.update(self._scan_udp_results)
 
   def _scan_tcp_ports(self):
-    max_port = 10000
     LOGGER.info('Running nmap TCP port scan')
     nmap_results = util.run_command( # pylint: disable=E1120
-        f'''nmap --open -sT -sV -Pn -v -p 1-{max_port}
+        f'''nmap --open -sT -sV -Pn -v -p 1-65535

```

Comment:
Full port scan will increase services module testing time significantly and negatively effect test times.  Full port scans should be done outside of Testrun and only the most common and ports of interest should be included in the automated portion because devices response time to port scans can't be reliably estimated.